signbank/dictionary/templates/dictionary/public_gloss_detail.html

```diff
@@ -48,7 +48,7 @@ <h2>
                 <header>
                     <h4>{% blocktrans %}Translation equivalents{% endblocktrans %}</h4>
                 </header>
-                {% for translation_language, translations in translation_languages_and_translations %}
+                {% for translation_language, translations in translation_languages_and_translations reversed %}
```

Comment:
Can you add a comment above this line please to explain why we reverse the translation languages? I presume by default it's ordered alphabetically by language name?